resources/PythonPip3Dsc/PythonPip3Dsc.psm1

```diff
@@ -83,7 +138,10 @@ function Get-PackageNameWithVersion {
         [string]$Version,
 
         [Parameter()]
-        [switch]$IsUpdate
+        [switch]$IsUpdate,
+
+        [Parameter()]
+        [switch]$DryRun
```

Comment:
I'm still not a fan of having function parameters defined that don't do anything. I would remove both IsUpdate and DryRun as they don't affect the behavior of this function. When you call this function, you can just explicitly use the parameters you need instead of passing the entire @PSBoundParameters

Reply:
Got it, can you run it again nitty Ryan ;)?